src/inventory-service/__mocks__/availability.ts

```diff
@@ -142,6 +145,7 @@ export const broadwayAvailabilityMock = {
             ],
             percentageDiscount: 0,
             includesBookingFee: true,
+            promotionLabel: 'Broadway promotion',
```

Comment:
I thought this one would be included in some test 🤔

Reply:
Me too, turns out it isn't used anywhere...